src/content-tags-drawer/TagBubble.scss

```diff
@@ -0,0 +1,9 @@
+.tag-bubble.btn-outline-dark:hover {
+  color: white;
+  background-color: #273F2F;
+  border-color: #273F2F;
+}
+
+.implicit > .btn-icon-before svg {
+  color: #E9E6E4; // $light-400;
```

Comment:
Why not just use $light-400?

Reply:
I tried using the variables like `$light-400` but it would not work. I kept getting errors in the build that it was not defined. I'm not sure what I am doing differently compared to the other existing scss files that utilized variables?

Reply:
You are importing `TagBubble.scss` into the `TagBubble` component [here](https://github.com/openedx/frontend-app-course-authoring/blob/12df9855c9b484b11ccdd72fdd61d1aa957cfd5b/src/content-tags-drawer/TagBubble.jsx#L7), which is fine, except that your `.scss` file doesn't import the SCSS variables nor Paragon base classes etc. And in fact you don't want to, because that will result in a double build and duplicated Paragon classes in the final CSS.

What you need to do instead is remove the reference to `TagBubble.scss` from your `.jsx` file, and instead include the custom styles via the master SCSS file, https://github.com/openedx/frontend-app-course-authoring/blob/master/src/index.scss . For simplicity, usually that file includes something from the top-level folder which then in turn includes the smaller .scss files.

---------

This is an "old-fashioned" system btw. The modern approach is to skip SCSS altogether and use CSS variables as well as relying much more heavily on Utility classes. I'm a big [Tailwind](https://tailwindcss.com/) fan and with that you have almost no styles at all in your global CSS and you have few or no separate CSS files. Generally you can achieve everything with utility classes like `d-flex align-self-start` etc. I don't think Paragon/boostrap's utility classes are as powerful, but the same principle applies - try to use utility classes as much as possible and create custom classes in the .css file only for very custom/unusual stuff where needed, like the Tag elements.

Reply:
oh I see - I followed your suggestions and it worked! I was getting confused as to why it wasn't working. Thanks for clearing it up for me.

Reply:
For what it's worth, once the design tokens bits land in paragon, the $light-400 will be replaced by a CSS variable anyway. There are automated tools to do that replacement in paragon alpha.